roles/openshift_prometheus/tasks/install_prometheus.yaml

```diff
@@ -0,0 +1,223 @@
+---
+
+# namespace
+- name: Add prometheus project
+  oc_project:
+    state: "{{ state }}"
+    name: "{{ openshift_prometheus_namespace }}"
+    node_selector: "{{ openshift_prometheus_node_selector | regex_replace(':','=') | regex_replace('{', '[') | regex_replace('}', ']') }}"
```

Comment:
The changes now allow the project to be created successfully. The role ran to completion.